docs/make-multi.jl

```diff
@@ -39,11 +39,10 @@ outpath = joinpath(@__DIR__, "build")
 MultiDocumenter.make(
     outpath,
     docs;
-    search_engine = MultiDocumenter.SearchConfig(),
-    # search_engine = MultiDocumenter.SearchConfig(
-    #     index_versions = ["stable"],
-    #     engine = MultiDocumenter.FlexSearch
-    # ),
+    search_engine = MultiDocumenter.SearchConfig(
+        index_versions = ["stable"],
+        engine = MultiDocumenter.FlexSearch
+    ),
```

Comment:
it would be good to remove the search entirely

Reply:
before we removed it, but the bar was still there. Perhaps the bar would be useful if versioning worked.